cfme/tests/configure/test_access_control.py

```diff
@@ -266,8 +266,10 @@ def test_user_edit_tag(appliance, group_collection):
     group = group_collection.instantiate(description=group_name)
 
     user = new_user(appliance, [group])
-    user.edit_tags("Cost Center *", "Cost Center 001")
-    assert ('Cost Center *', 'Cost Center 001') in user.get_tags(), "User edit tag failed"
+    user.add_tag(category="Cost Center *", tag="Cost Center 001")
+    assert (
+        tag.category.display_name == 'Cost Center *' and tag.display_name == 'Cost Center 001'
```

Comment:
This logic isn't correct, you need to wrap the generator in an all() call.

```python
assert all([tag.category.display_name == 'Cost Center *' and tag.display_name == 'Cost Center 001'
                 for tag in user.get_tags()]), 'Assigned tag was not found'

Reply:
In this this case I think @rlbabyuk is trying to check if any of the tags have these particular attributes.   So as you mentioned offline any() needs to be used:
    assert any([
        tag.category.display_name == 'Cost Center *' and tag.display_name == 'Cost Center 001'
        for tag in user.get_tags()
    ]), 'Assigned tag was not found'

Reply:
Actually, you can omit the `[]`s in this case.

Reply:
mfalesni, interesting, noted. I think I'd prefer to keep them in the interest of being explicit rather than implicit.